.github/workflows/package_deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Package and Deploy
+
+on:
+  push:
+    branches:
+      - feature/package-artifact
+  workflow_dispatch:
+
+
+jobs:
+  build_package_deploy:
+    name: Deploy Module
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup PowerShell module cache
+        id: cacher
+        uses: actions/cache@v3
+        with:
+          path: "~/.local/share/powershell/Modules"
+          key: ${{ runner.os }}-PSModules
+      - name: Setup
+        shell: pwsh
+        run: |
+          ./Tools/setup.ps1
+          Invoke-Build -Task ShowInfo
```

Comment:
consider removing this, if we don't need it.
I added this back in the day because the information was important to debug failed builds

Reply:
I find it useful and doesn't impact negatively the process. But your call.